WcaOnRails/app/assets/javascripts/persons.js

```diff
@@ -1,6 +1,10 @@
 // Add params from the search fields to the bootstrap-table for on Ajax request.
 var personsTableAjax = {
   queryParams: function(params) {
+    if (personsTableAjax.queriesOK !== true) {
```

Comment:
Yeah, I went back and forth on that, but in the end I slightly prefer this.

I find it hard to instantly see the difference between `if (long_name) ...` and `if (!long_name) ...`, so I tend to avoid the `!`. That also why I love `unless` and have tried to like `not` in Ruby.

Anyway, I'll merge it up tomorrow.